cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2491,6 +2496,21 @@ impl MachInst for Inst {
         }
     }
 
+    fn is_included_in_clobbers(&self) -> bool {
```

Comment:
Similar naming concern here.  Also, for good performance, this needs to be inlined, no?

Reply:
Verified that inlining occurs; rustc seems to be doing the right thing here. (It's statically monomorphized so I expect it always would as long as someone does not inflate the body to some huge size...).

Reply:
Re: naming: were you thinking something more specific to calls, as above, or something else?

I put a bit of thought into this and I'm not sure that I like including "call instruction", as the concept is more general. (In principle the `MachInst` layer doesn't even know about the concept of a call instruction.) The concept really is just "should we aggregate this instruction's effects into the summary of clobbered registers". Any other suggestions?

Or if we do decide that this is specific to calls, I think I'd prefer `is_call_instruction_with_ignored_clobbers()`. I could go with this; it just seems a bit non-orthogonal, is all.

Reply:
Ah, leave it as it is then.  That the other names have been improved is good enough for me.  Thanks for doing that.